ui/src/app/features/user-list/user-list.component.ts

```diff
@@ -101,10 +101,7 @@ export class UserListComponent implements OnInit, OnDestroy {
               data: {
                 entityType: this.translate.instant('users.user'),
                 entity: deletion.userToDelete,
-                options: [
-                  { name: email, value: 'deleter' },
-                  { name: this.orgOwnerEmail, value: 'owner' },
-                ],
+                options: { name: this.orgOwnerEmail, value: 'owner' },
```

Comment:
displaying 2 emails in the modal was a requirement. Please confirm it is no longer needed.

Reply:
@isobolev confirmed with QA (Anastasiya Arutsiunian). She said that we no longe have situation with 2 emails display